code/modules/antagonists/roguetown/villain/bandit.dm

```diff
@@ -10,7 +10,6 @@
 		"I WILL NOT LIVE IN YOUR WALLS!",
 		"I WILL NOT FOLLOW YOUR RULES!",
 	)
-	rogue_enabled = TRUE
```

Comment:
What does this do?

Reply:
as far as i know `rogue_enabled` defines whether the antag is visible in the Antag panel helps avoiding giving people blob antag and shit.

could have other uses.